src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionStand.java

```diff
@@ -9,6 +9,20 @@
 @Cacheable
 @org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
 public class SelektionStand extends SelektionHierarchy {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID")
+    private Integer id;
```

Comment:
Adding an ID field here seems to be wrong, since SelektionStand inherits from SelektionHierarchy > SelektionBezeichnung > Selektion, and the id column as well as the getter is already defined in the Selektion class. (and adding a setter should not be necessary)

Reply:
is removed